presto-docs/src/main/sphinx/connector/blackhole.rst

```diff
@@ -0,0 +1,25 @@
+====================
+Black Hole Connector
+====================
+
+The Black Hole connector works in similar way as /dev/null device file in Linux operating system. 
```

Comment:
As a non native English speaking person it is difficult for me to do the 'wording'. Also I don't know what do you kind of formatting you mean. Do you have some rules how documentation should be written?

It would be much easier for me if you would point me out  each place where something have to be fixed. Such general comments don't help me much ; /

I looked how Cassandra connector is documented, and I try to follow its convention of formatting. I also added a Configuration section.